inductiva/storage/README.md

```diff
@@ -0,0 +1,46 @@
+# Storage Management
+
+The Inductiva API provides you with the tools to manage your cloud storage effectively. Currently, we support Google Cloud, but we're working on expanding our services to include multiple cloud platforms soon!
+With the Inductiva storage module, you can easily navigate your storage bucket, evaluate the space used, and delete specific directories as needed.
```

Comment:
Don't mention bucket, since it is specific of Google Cloud - I think *your storage is enough.